libqtile/widget/camcheck.py

```diff
@@ -0,0 +1,43 @@
+# A qtile widget written to show active indication for mic or camera
+# written by bhauvk sharma (https://github.com/desdemona2)
+
+from libqtile.widget import base
+import os
+
+
+class active(base.ThreadPoolText):
+    """widget to show indication if
+    camera and mic are active in status bar"""
+    
+    defaults = [
+           (
+            "update_interval", 
+            1, 
+            "Update interval in seconds, if none, the "
+            "widget updates whenever it's done'."
+            ),
+    ]
+
+
+    def __init__(self, **config):
+        super().__init__("", **config)
+        self.add_defaults(active.defaults)
+
+
+    def poll(self):
+        camera = os.system("fuser /dev/video0")
```

Comment:
The devices should be configurable as users may have multiple devices.

Reply:
Oh my bad never thought of this and SORRY i completely forgot i had a pending Commit in this repo.